tools/gcs-lock/README.md

```diff
@@ -0,0 +1,29 @@
+## GCS Lock
```

Comment:
Do we plan on distributing this as a Maven package? If not, let's put this in the `examples/` directory.

Reply:
Moved to `examples/`